source/isaaclab_mimic/isaaclab_mimic/locomanipulation_sdg/envs/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2024-2025, The Isaac Lab Project Developers (https://github.com/isaac-sim/IsaacLab/blob/main/CONTRIBUTORS.md).
+# All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Sub-package with environment wrappers for Locomanipulation SDG."""
+
+import gymnasium as gym
+
+from .g1_locomanipulation_sdg_env import G1LocomanipulationSDGEnvCfg
```

Comment:
Please avoid importing the environment in there directly. It increases the import time of all dependencies.

You can register the environment without it as shown here: https://github.com/isaac-sim/IsaacLab/blob/main/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/anymal_b/__init__.py#L13-L23

Reply:
Fixed using your suggestion